.eslintrc.cjs

```diff
@@ -14,6 +14,7 @@ module.exports = {
     plugins: ['react-refresh'],
     rules: {
         'react/jsx-no-target-blank': 'off',
-        'react-refresh/only-export-components': ['warn', { allowConstantExport: true }]
+        'react-refresh/only-export-components': ['warn', { allowConstantExport: true }],
+        "prettier/prettier": "off" 
```

Comment:
You can't do this you are not allow to change this file

Reply:
i have done this by mistake sir i am resolving this right now